index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="portfolio.css">
+    <title>Nsanzimfura portfolio</title>
+
+</head>
+<body>
+    <header>
+        <nav class="top-navigation-bar">
+            <h2 class="header-welcome">Welcome</h2>
+            <img class="menu-icon" src="./assets/header-files/icon-left-side-menu.svg" alt="menu icon" />
+        </nav>
+    </header>
+    <main>
+        <section class="main-page">
+            <p class="introduction-text"><span class="dark-blue">Hey there. I'm <br>Eden<br></span>
```

Comment:
Good work creating a section and wraping the elements inside it ❗❗❗.

- [ ] For better practice and semantic tags, please change this tag `p` for an `h1` tag, since this is the title of the section and for accessibility readability will be more helpful for devices with a tab navigation 

If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.